apollo-federation/src/schema/type_and_directive_specification.rs

```diff
@@ -957,7 +946,11 @@ mod tests {
             false,
             &[DirectiveLocation::Object],
             true,
-            Some(Box::new(link_spec)),
+            Some(&|_| {
+                LINK_VERSIONS
+                    .find(&Version { major: 1, minor: 0 })
+                    .map(|v| v as &'static dyn SpecDefinition)
```

Comment:
`static` could be dropped.
```suggestion
                    .map(|v| v as &dyn SpecDefinition)
```